src/ImageStack.cpp

```diff
@@ -184,20 +242,26 @@ void ImageStack::generateMask() {
         for (size_t y = 0; y < height; ++y) {
             for (size_t x = 0; x < width; ++x) {
                 size_t i = 0;
-                while (i < images.size() - 1 &&
-                    (!images[i].contains(x, y) ||
-                    images[i].isSaturatedAround(x, y))) ++i;
+                while (i < images.size() - 1 && (!images[i].contains(x, y) ||
+                    images[i].isSaturatedAround(x, y))) {
+                    // skip if the pixel is not in image or clipped
+                    ++i;
+                }
                 mask(x, y) = i;
             }
         }
     }
-    // The mask can be used in compose to get the information about saturated pixels
+    // The mask can be used in compose() to get the information about saturated pixels
     // but the mask can be modified in gui, so we have to make a copy to represent the original state
-    origMask = mask;
+    original_mask = mask;
 }
 
 
 double ImageStack::value(size_t x, size_t y) const {
+    /*
```

Comment:
Now that I realize it: I'd prefer C++ comments. They are easier to maintain.

Reply:
What do you mean by C++ comments? is it this?
``` C++
/* 
I am a multi line
comment
*/
```
instead of what I currently did, which is:
``` c++
/*
 * This method will approximate a saturation threshold for the brightest
 * image only. It is sufficient because the calculated threshold will
 * b
 */
```

I simply copied what every files starts with:
``` c
/*
 *  HDRMerge - HDR exposure merging software.
 *  ...
 */
```

Sorry I simply lack knowledge in C/C++, a quick google search gives mixed results.

Which comment technique do you recommend using?

Reply:
Sorry for the confusion. I meant C++-style comments like

```c++
// This is a multi line comment
// which can be easily edited and
// aligned.
// Some linters have problems with
// fancy C-style comments (e.g. uncrustify).
```

But it's absolutely up to you. If you feel like it's more consistent the way it is now, then better keep it that way. :+1:

Reply:
Just adding more info here for future readers: 
- i will use C++ style comments for explaining lines or small parts 
- i will use the `/**/` block comment style for methods/classes description because they are foldable in most text editors meaning less distraction for the programmer